modules/Blockchain/Ethereum/index.js

```diff
@@ -82,10 +83,27 @@ class Ethereum {
     }
 
     /**
-     * Initializes Blockchain provider (get contract addresses, etc.)
+     * Reads identity from file
+     * @returns {Promise<erc725Identity>}
+     * @private
+     */
+    _loadIdentityFromFile() {
```

Comment:
This has been moved to the utilities class

Reply:
Okay, will fix in the next PR, as not to block this one too much